main.tf

```diff
@@ -9,51 +9,31 @@ resource "google_container_cluster" "cluster" {
 
   project = local.project
 
+  enable_autopilot = true
+
   network    = var.network
   subnetwork = var.subnetwork
 
   name            = var.name
   description     = var.description
   resource_labels = var.resource_labels
 
-  location       = var.location
-  node_locations = var.node_locations
+  location = var.location
 
-  networking_mode             = var.networking_mode
-  enable_intranode_visibility = var.enable_intranode_visibility
-  private_ipv6_google_access  = var.private_ipv6_google_access
+  networking_mode            = var.networking_mode
+  private_ipv6_google_access = var.private_ipv6_google_access
 
-  enable_shielded_nodes       = var.enable_shielded_nodes
-  enable_binary_authorization = var.enable_binary_authorization
-  enable_kubernetes_alpha     = var.enable_kubernetes_alpha
-  enable_tpu                  = var.enable_tpu
-  enable_legacy_abac          = var.enable_legacy_abac
+  enable_kubernetes_alpha = var.enable_kubernetes_alpha
+  enable_tpu              = false
+  enable_legacy_abac      = false
 
   # TODO: use data source to allow fuzzy version specification
   min_master_version = var.min_master_version
 
   logging_service    = var.logging_service
   monitoring_service = var.monitoring_service
 
-  cluster_ipv4_cidr         = var.cluster_ipv4_cidr
-  default_max_pods_per_node = var.default_max_pods_per_node
-
-  dynamic "authenticator_groups_config" {
-    for_each = var.rbac_security_identity_group != null ? [1] : []
-
-    content {
-      security_group = var.rbac_security_identity_group
-    }
-  }
```

Comment:
i assume we still want to use google groups for rbac? why remove it? this is still an optional feature in autopilot

Reply:
As discussed, I will open an issue in https://www.google.com/search?q=github+google+provider&rlz=1C5CHFA_enDE764DE764&oq=github+google+provider&aqs=chrome..69i57j35i39j0i512l2j69i60l3j69i65.4299j0j7&sourceid=chrome&ie=UTF-8